runhouse/resources/hardware/cluster.py

```diff
@@ -739,18 +739,37 @@ def connect_server_client(self, force_reconnect=False):
                 system=self,
             )
 
-    def status(self, resource_address: str = None):
-        """Loads the status of the Runhouse daemon running on the cluster."""
+    def status(self, resource_address: str = None, send_to_den: bool = False):
+        """Load the status of the Runhouse daemon running on a cluster."""
+
+        # TODO [sb]: add to docs the response schema of the status output, so users will know what to expect.
         # Note: If running outside a local cluster need to include a resource address to construct the cluster subtoken
         # Allow for specifying a resource address explicitly in case the resource has no rns address yet
         if self.on_this_cluster():
-            status = obj_store.status()
+            status, send_to_den_resp = obj_store.status(send_to_den=send_to_den)
         else:
-            status = self.call_client_method(
+            status, send_to_den_resp = self.call_client_method(
                 "status",
                 restart_server=False,
                 resource_address=resource_address or self.rns_address,
+                send_to_den=send_to_den,
             )
+
+        if send_to_den:
+            send_to_den_status_code = send_to_den_resp.get("status_code")
```

Comment:
if we return  the resp object here, we can call `resp.status_code`, and then in the warning message also just do `resp.json()`